lessons/Algebra2/exploring-quadratic-models/langs/en-us/pages/graphing-models.adoc

```diff
@@ -0,0 +1,62 @@
+= Graphing Quadratic Models
+++++
+<style>
+.autonum { font-weight: bold; }
+.autonum:after { content: ')' !important; }
+
+.FillVerticalSpace { grid-gap: 20px; margin-top: 5px; }
+
+td {
+	background-color: rgba(#fff, .5);
+	background-image:
+		linear-gradient(lightblue 2px, transparent 2px),
+		linear-gradient(90deg, lightblue 2px, transparent 2px),
+		linear-gradient(rgba(150, 150, 150, .3) 1px, transparent 1px),
+		linear-gradient(90deg, rgba(150, 150, 150, .3) 1px, transparent 1px);
+	background-size: 100px 100px, 100px 100px, 20px 20px, 20px 20px;
+	background-position: -3px 74px;
+	min-height: 1.5in;
+	max-width: 2in;
+}
+</style>
+++++
+
+For this page, you'll need to have @online-exercise{https://www.desmos.com/calculator/geyx4aetop, the Exploring Quadratics Desmos Activity} open on your computer. The parabola you see is the graph of @math{f(x)=x^2}. Another, *identical* parabola is hiding behind it. This second parabola is written in Vertex Form: @math{g(x)=a(x-h)^2+k}. Each coefficient starts at values to make @math{g(x)} equivalent to @math{f(x)}.
+
+@n Use the values of @math{a}, @math{h}, and @math{k} to write the Vertex Form of @math{x^2}: @fitb{}{@ifsoln{@math{1(x-0)^2+0}}}
+
+== Magnitude @math{a}
+@n Try changing the value of @math{a} to -4, 0, and 2, graphing each parabola in the squares below and labeling the vertex and roots.
+
```

Comment:
perhaps we want to have them label something other than the vertex and the roots since all of these parabolas have a minima at the origin and zero precision is required in their graphs if that's what we ask of them? or at least we could label the axes of the grid so that the scale stays the same, forcing them to be precise with their plots?

Reply:
There are already grids here. Is it just that you want me to add labels for the axes?

Reply:
Yes. Because it's hard to see patterns when the axes are inconsistent... and eventually the parabola will both flip upside down and move up and down, which students are unlikely plan for when they don't know it's coming.

Reply:
Ooooh, this one was a _serious_ CSS challenge! Done.